src/core/probe/kernel/bpf/include/helpers.h

```diff
@@ -3,8 +3,7 @@
 
 #include <bpf/bpf_tracing.h>
 
-extern int LINUX_KERNEL_VERSION __kconfig;
-extern bool CONFIG_X86_KERNEL_IBT __kconfig __weak;
+enum bpf_func_id___x { BPF_FUNC_get_func_ip___x = 42 };
```

Comment:
not related to this change (which is ok), but more a broader remark. We should probably try to "establish some guidelines" for this kind of suffix.
Assuming it makes sense, we can consolidate this in a follow-up.

Reply:
I guess that ATM we have the nft module that does that plus this new addition.
Initially, in the nft module ___retis was used, but it's a bit redundant and perfectly equivalent to ___x as the triple underscore is IMO self-explanatory, and "retis" doesn't add anything.
Eventually, the kernel version that introduced the type was used. The downside of this is that it could be not straightforward (the version has to be retrieved).
Not sure there's a common good practice here, but having a uniform suffix could be helpful for consistency, assuming other triple underscore suffixes will be used.

Reply:
ok. sgtm